src/components/Results/Submission.vue

```diff
@@ -138,9 +138,11 @@ export default {
 			font-weight: bold;
 		}
 
```

Comment:
```suggestion
		}
```
Remove extra new line